Resources/config/auto_route.xml

```diff
@@ -86,6 +86,14 @@
             class="%cmf_routing_auto.metadata.factory.class%"
             factory-service="cmf_routing_auto.metadata.factory.builder"
             factory-method="getMetadataFactory"
-        />
+            />
+
+        <!-- Controller -->
+        <service 
+            id="cmf_routing_auto.redirect_controller"
+            class="Symfony\Cmf\Bundle\RoutingAutoBundle\Controller\RedirectController"
```

Comment:
i think i read that in a blogpost on symfony.com but did not find it anymore. there is this https://github.com/symfony/symfony/issues/11881